lib/pageflow/configuration/permissions.rb

```diff
@@ -0,0 +1,30 @@
+module Pageflow
+  class Configuration
+    # Permissions related options to be defined in the pageflow
+    # initializer of the main app.
+    class Permissions
+      def initialize(options = {})
+        @show_entry_features_to_non_admin =
+          options.fetch(:show_entry_features_to_non_admin, true)
+        @show_admin_boolean_to_non_admin =
+          options.fetch(:show_admin_boolean_to_non_admin, true)
+        @show_theming_dropdown_to_non_admin =
+          options.fetch(:show_theming_dropdown_to_non_admin, true)
+      end
+
+      # Allow non-admin users to see entry features. Defaults to true.
+      # @since edge
+      attr_accessor :show_entry_features_to_non_admin
```

Comment:
Given that letting account managers edit features will be the default, we could also consider negating the name: `only_admins_may_update_features`. That way it would be clear that these options can be used to restrict permissions above what is the default.